modules/claims_api/spec/requests/v0/claims_request_spec.rb

```diff
@@ -70,13 +70,36 @@
         expect(response).to match_response_schema('claims_api/claim')
       end
     end
+
     context 'with errors' do
       it '404s' do
         VCR.use_cassette('evss/claims/claim_with_errors') do
           get '/services/claims/v0/claims/123123131', params: nil, headers: request_headers
           expect(response.status).to eq(404)
         end
       end
+
+      it 'shows a single Claim through auto established claims with an error', run_at: 'Wed, 13 Dec 2017 03:28:23 GMT' do
+        create(:auto_established_claim,
+               auth_headers: { some: 'data' },
+               evss_id: 600_118_851,
+               id: 'd5536c5c-0465-4038-a368-1a9d9daf65c9',
+               status: 'errored',
+               evss_response: { 'messages' => [{ 'key' => 'serviceError', 'severity' => 'FATAL', 'text' => 'Not established.' }] })
+        VCR.use_cassette('evss/claims/claim') do
```

Comment:
Does this warrant a factory attribute or brand new factory altogether? Will there be lots of work on AEClaims going forward? 

I'm indifferent until there's more duplication

Reply:
We can eventually cleanup and move this into a factory but it feels out of scope for now considering we are transitioning over teams and just need to close this out.